crates/swc_css_minifier/tests/fixture/compress-calc/ignore-media-query/input.css

```diff
@@ -0,0 +1,2 @@
+@media (min-width:calc(10px+10px)){
+}
```

Comment:
To avoid this please add `color: red` for all tests where simple block is empty (maybe we should add option to disable it for better testing, but let's do it late)

Reply:
Fixed